Src/ILGPU.Algorithms/Runtime/Cuda/API/CuFFTAPI.xml

```diff
@@ -0,0 +1,276 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ImportLibrary Namespace="ILGPU.Runtime.Cuda.API"
+               ClassName="CuFFTAPI"
+               VersionType="CuFFTAPIVersion"
+               DefaultDllImportSearchPath="LegacyBehavior"
+               DefaultMethodAccessModifier="public"
+               DefaultReturnType="CuFFTResult">
+    <LibraryVersions>
+        <LibraryVersion Name="V10" Platform="Windows" LibName="cufft64_10.dll" />
+        <LibraryVersion Name="V10" Platform="Linux" LibName="libcufft.so.10" />
+        <LibraryVersion Name="V10" Platform="OSX" LibName="libcufft.10.dylib" />
```

Comment:
QUESTION: Are these the correct file names for Linux and OSX?

Reply:
The library names seem to be correct 👍